assets/js/modules/adsense/datastore/tags.js

```diff
@@ -326,7 +342,7 @@ export const selectors = {
 	 *
 	 * @param {Object} state    Data store's state.
 	 * @param {string} clientID The AdSense Client ID to check permissions for.
-	 * @return {?boolean} True if the user has access, false if not; `undefined` if not loaded.
+	 * @return {boolean|undefined} True if the user has access, false if not; `undefined` if not loaded.
```

Comment:
Multiple return types should be wrapped in parens for JSDoc:
```suggestion
	 * @return {(boolean|undefined)} True if the user has access, false if not; `undefined` if not loaded.
```